cmd/image/import/import.go

```diff
@@ -251,8 +251,13 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME>  --object rhel-83-100
 }
 
 func init() {
+	// TODO pvs-instance-name and pvs-instance-id is deprecated and will be removed in a future release
 	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.InstanceName, "pvs-instance-name", "n", "", "PowerVS Instance name.")
+	Cmd.Flags().MarkDeprecated("pvs-instance-name", "pvs-instance-name is deprecated, pvs-workspace-name should be used")
 	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.InstanceID, "pvs-instance-id", "i", "", "PowerVS Instance ID.")
+	Cmd.Flags().MarkDeprecated("pvs-instance-id", "pvs-instance-id is deprecated, pvs-workspace-id should be used")
```

Comment:
any specific reason for making this as `pvs-workspace-id` instead of just `workspace-id`?

Reply:
I am not sure on this, I renamed this `pvs-instance-id` variable. I will update the variables to `workspace-id`.

Reply:
okay, lets take this opportunity to keep the variable same across all the commands

Reply:
Sure, updated the variable